static/src/javascripts/projects/common/modules/commercial/dfp/apply-creative-template.js

```diff
@@ -75,10 +75,8 @@ define([
         var creativeConfig = fetchCreativeConfig();
 
         if (creativeConfig) {
-            return Promise.all([
```

Comment:
That the creative has been rendered, which is why the promise has to resolve with whatever `renderCreative` returns, instead of an array. The iframe is not really part of the rendering, is it?